### DISCUSSION TITLE
What does the custom shortcut key run mean?

### DISCUSSION THREAD

**Discussion by alaric8** [gdh1995/vimium-c#1216] [Q&A]
![image](https://github.com/user-attachments/assets/604bddee-f6b0-40ca-a063-828c9f6353c6)

**Comment by gdh1995** [comment]
Its a grammar sugar and equal with `map Cd runKey keys="Cv" keyword="dict"`

**Reply by alaric8** [reply]
Okay, thank you. Clear up the confusion.
